qutebrowser/app.py

```diff
@@ -255,7 +256,11 @@ def process_pos_args(args, via_ipc=False, cwd=None):
             log.init.debug("Empty argument")
             win_id = mainwindow.get_window(via_ipc, force_window=True)
         else:
-            win_id = mainwindow.get_window(via_ipc)
+            if via_ipc and target_arg and target_arg != 'auto':
```

Comment:
With the `and target_arg` you're checking if it's not the default argument, right? If so, please use `and target_arg is not None` instead.